client/public/src/components/app.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from 'react';
+import ReactDOM from 'react-dom';
+import axios from 'axios';
+
+import Details from './details.jsx';
+import Options from './options.jsx';
+
+const App = () => {
+  const [productId, setProductId] = useState(1);
+  const [product, setProduct] = useState({});
+  const [storeId, setStoreId] = useState(1);
+  const [stores, setStore] = useState([]);
```

Comment:
I think it's best practice to have these be consistent (`store` vs. `stores`)

Reply:
I made it so things that are plural have a collection of things such as stores having all the stores, while things like product have a single item, the selected product.

Reply:
oh nevermind I see what you mean, stores, and setStore. Thanks!